README.md

```diff
@@ -1 +1,131 @@
 # Hub Chain PoC
+
+Proof-of-Concept implementation of a Hub Chain based on Hyperledger Iroha 2.
+
+The core idea of a hub chain is to move assets across Iroha 2 blockchains (called _domestic chains_) using a **central hub chain**.
+
+The transfer is split in two steps:
+
+1. Chain 1 → Hub Chain
+2. Hub Chain → Chain 2
+
+Each step involves a **prover** (source chain) and a **verifier** (target chain). However, for now we will be implementing a simple flow without proofs.
+
+## Stage 1 _(you are here)_: No-proof
+
+In this version, there is no prover-verifier mechanism. Cross-chain transfers are done by the trusted relays.
+
+This lacks the core requirement for the hub chain, but may be a good preparation to introduce a prover-verifier mechanism later based on this work (even if there will be drastic design changes).
+
+### Design
+
+- There are $N$ domestic chains and 1 hub chain,
+- Each _domestic_ chain has an arbitrary amount of user accounts, $N - 1$ _omnibus_ accounts, and 1 _relay_ account
+- Hub chain has $N$ omnibus accounts and $N$ relay accounts
+- _All_ chains have the same numeric asset definitions (e.g. `rose#wonderland`, `time#looking_glass` etc)
+- Balances of omnibus accounts are representing totals of their associated chain in order. If some asset is minted on some chain, it must be minted to the omnibus account of this chain on all other chains (both domestic and the hub).
```

Comment:
In actual use cases, changes in the total supply (mint/burn) occur only on the hub chain, and each domestic chain follows accordingly.